src/client/utils/dataset/dataset.ts

```diff
@@ -17,12 +17,14 @@
 
 import { Timezone } from "chronoshift";
 import { Dataset, Datum, NumberRange, TimeRange } from "plywood";
+import { SortDirection } from "../../../common/models/sort/sort";
+import { Split, SplitType } from "../../../common/models/split/split";
 import { formatValue } from "../../../common/utils/formatter/formatter";
 import { SPLIT } from "../../config/constants";
 
-export type Order<D> = (a: [string, number, D], b: [string, number, D]) => number;
+type Order<D> = (a: [string, number, D], b: [string, number, D]) => number;
```

Comment:
What kind of sorcery is this? Where the third element of the tuple goes in `orderByValueDecreasing` definition? Is it simply omitted and that's just fine for TS?

Reply:
It is in type definition via `Order` type. In Implementation we don’t use it so we don’t destructure for it. It’s more like ... JavaScript :)